packages/langium/src/lsp/langium-lsp-module.ts

```diff
@@ -0,0 +1,115 @@
+/******************************************************************************
```

Comment:
I would prefer the name `default-lsp-module` to emphasize that it contributes defaults for all languages, not just the Langium grammar language (could be perceived as such)
(and to keep the name similarity to the `default-module`)

Reply:
I'm not opposed to this, but I imagine we should discuss this together. I have a feeling we might be renaming the default module to a 'core' module, which would change the case.

Reply:
Came back to this and decided to apply it, change will be up in a moment.